azuredisk-csi.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: azuredisk-csi
+  version: 1.31.0
+  epoch: 0
+  description: Azure Disk CSI Driver
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 81f73ae6e758c2dd5efbaf127014e80d01d39f66
+      repository: https://github.com/kubernetes-sigs/azuredisk-csi-driver
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.driverVersion=v${{package.version}}
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.gitCommit=$(git rev-parse HEAD)
+        -X sigs.k8s.io/azuredisk-csi-driver/pkg/azuredisk.buildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        -extldflags "-static"
+      output: azurediskplugin
+      tags: azurediskv2
+      packages: ./pkg/azurediskplugin
+
+  - uses: strip
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/azurediskplugin ${{targets.subpkgdir}}/azurediskplugin
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes-sigs/azuredisk-csi-driver
+    strip-prefix: v
+    tag-filter: v
+    use-tag: true
```

Comment:
the project is doing active releases, maybe we can use releases too. This way we'll exclude random tags if this ever gets pushed. very unlikely given it's a kubernetes-sigs project but maybe some tags to track helm chart releases or something.